lib/ViewModels/SearchTabViewModel.js

```diff
@@ -49,8 +49,11 @@ var SearchTabViewModel = function(options) {
 
     // Focus the search box when the tab is activated.
     knockout.getObservable(this, 'isActive').subscribe(function(newValue) {
-        if (newValue) {
```

Comment:
This is probably not necessary but i added for consistency, maybe not a good idea?